etc/checksum.frontend

```diff
@@ -1,5 +1,5 @@
 ###################################################################
-# GLIDEINWMS_VERSION v3_10_7_RC3
+# GLIDEINWMS_VERSION v3_7_10
```

Comment:
version should be `3.10.7`

Reply:
Resolving this for now. 

As per Marco: the RPM and content for 3.10.7 are good to go. It is the tarball name and git tag that reflects the version as 3.7.10. Ignoring this for now as once this PR gets approved and merged, `3.10.7-2` will be done with the correct tag.